app/scripts/main.js

```diff
@@ -74,15 +74,26 @@ $(document).ready(function () {
         };
     };
 
-    var openManPage = function () {
+    var openManPage = function (selectedManPage) {
         var openManPageTabs;
-        if (lastSelectedManPage) {
+        if (selectedManPage) {
             openManPageTabs = viewModel.openManPageTabs;
-            openManPageTabs.push(getManPageTabData(lastSelectedManPage));
+            openManPageTabs.push(getManPageTabData(selectedManPage));
             viewModel.currentActiveTabIndex(openManPageTabs().length - 1);
         }
     };
 
+    var manPageTokens = new Bloodhound({
+        datumTokenizer: Bloodhound.tokenizers.obj.whitespace('name', 'summary'),
```

Comment:
It was supposed to be only name, that was the entire reason behind creating a new bloodhound reason so that it only tokenizes name. I guess I forgot to include it in the initial commit